api/v1/models/User.js

```diff
@@ -16,7 +16,8 @@ const User = Model.extend({
   hasTimestamps: ['created', 'updated'],
   validations: {
     email: ['required', 'email'],
-    password: ['required', 'string']
+    password: [ 'string' ],
+    githubId: [ 'string' ]
```

Comment:
If this is their GitHub handle, you might consider calling it `githubHandle`. Calling it `githubId` makes it sound numeric.